documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -117,9 +102,29 @@ You can add source providers and destination providers for a virtual machine mig
 [id="adding-source-providers"]
 ==== Adding source providers
 
-You can add a VMware source provider, {a-rhv} source provider, or an {osp} source provider by using the {ocp} web console.
+You can use {project-short} to migrate VMs from the following source providers:
+
+* {virt}
+* {osp}
+* {rhv-full}
+* VMware vSphere
+* Open Virtual Appliances (OVAs) that originate from or are compatible with VMware vSphere.
+
+You can add a source provider by using the {ocp} web console.
 
 :mtv!:
+:context: cnv
+:cnv:
+include::modules/adding-source-provider.adoc[leveloffset=+4]
+:cnv!:
+:context: ostack
+:ostack:
+include::modules/adding-source-provider.adoc[leveloffset=+4]
```

Comment:
should be `osh-adding-source-provider`, no?

Reply:
@ahadas -- Ignore this -- this is the new structure of master.adoc based on the new Create Providers PR. I changed master.adoc to try to reduce future merge conflicts. Nothing in this PR affects the procedures for creating the source provider for OpenStack